src/test/java/uk/gov/ch/model/psc/PersonWithSignificantControlTest.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.ch.model.psc;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.companieshouse.api.model.common.Address;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class PersonWithSignificantControlTest {
+
+    private static String HOUSE_NAME_NUMBER = "19";
+    private static String STREET = "My Street";
+    private static String CARE_OF = "care of";
+    private static String COUNTRY_NAME = "England";
+    private static String POST_TOWN = "London";
+    private static String PO_BOX = "PO box";
+    private static String POST_CODE = "LW1 1AA";
+    private static String REGION = "region";
+
+    @Test
+    void testUraAddress() {
+        PersonWithSignificantControl psc = new PersonWithSignificantControl();
+        psc.setHouseNameNumber(HOUSE_NAME_NUMBER);
```

Comment:
Minor, but might be worth setting something different to the 'house name number' value on the 'addressLine1' field just to be sure it doesn't somehow (now or in the future) end up in the 'address line 1' of the address.